gh_actions_exporter/cost.py

```diff
@@ -0,0 +1,123 @@
+import jinja2
+
+from typing import Dict, List
+from gh_actions_exporter.config import Relabel, RelabelType, Settings
+from gh_actions_exporter.types import WebHook, JobCost
+from gh_actions_exporter.tokenGenerator import TokenGenerator
+from github import Github, Repository
+
+class Cost(object):
```

Comment:
while having a cost class is a good idea, not all functions are related to cost. Some are merely relabelling functions.

Reply:
I think now it's better, we only have `_get_job_cost` & `_runner_type_job` not related this much

Reply:
Tell me if it's ok for you now, otherwise tell me what you want for me to do